source/operational-efficiency.txt

```diff
@@ -7,11 +7,35 @@ Operational Efficiency
 
 .. default-domain:: mongodb
```

Comment:
On lines 1-2 above, can we replace the meta robots with :noprevnext:? I think now that we have content here we can index the pages. This applies for all of these former drawer pages.